protostar-rust/src/pretty_printing.rs

```diff
@@ -24,12 +24,21 @@ pub fn print_running_tests(test_file: &Utf8PathBuf, tests_num: usize) {
 }
 
 pub fn print_test_summary(tests_stats: TestsStats) {
-    println!(
-        "{}: {} passed, {} failed",
+    let skipped_str = if tests_stats.skipped > 0 {
+        format!(", {} skipped", tests_stats.skipped)
+    } else {
+        String::new()
+    };
```

Comment:
Can we please print it every time? We are trying to be consistent with Cargo behaviour. Please remember to change tests accordingly.